stats/ViewConfig.md

```diff
@@ -0,0 +1,126 @@
+# View Definition
+
+This document describes the YAML configuration for views.
+
+Implementations should provide a way to load this configuration to define views.
+Implementations may provide facilities to:
+* watch the configuration file for changes and reload the views
```

Comment:
Is the file here any resource? E.g. metadata from a metadata service?

I'd not call it file to be more precise.